### ISSUE TITLE
Does SCRFD generalize?

### ISSUE THREAD

**Issue by tomchen1000** [deepinsight/insightface#1580]
The SCRFD is very interesting. But I have the following concern:

It explores using the WIDER Face validation set and reports results on the validation set. It might overfit the validation set. 

Does SCRFD generalize? Or does it generalize to the WIDER Face test set? 

Thanks!
